src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,97 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float DEFAULT_LOAD_FACTOR = 0.75F;
+    private static final int INCREASE = 2;
```

Comment:
Replace INCREASE to INCREASE_INDEX and refactor code where this variable was called. As for me its more informative 
```suggestion
    private static final int INCREASE_INDEX = 2;
```